src/media/js/addon/containers/submit.js

```diff
@@ -4,6 +4,8 @@ import {bindActionCreators} from 'redux';
 import FileReaderInput from 'react-file-reader-input';
 import {connect} from 'react-redux';
 
+import AddonSubnav from '../components/addonSubnav';
+import PageHeader from '../../site/components/pageHeader';
```

Comment:
i'd move this down one line. the way i'm ordering imports is by the latter section. so it would start importing from the current directory, then the parent, and so on, all alphabetically.